src/scripts/const.ts

```diff
@@ -0,0 +1,66 @@
+export const OMNIBRIDGE_CONTRACT_ABI = [{
```

Comment:
nitpick, maybe better to add it to a `src/abi` instead of `src/scripts/const`?

Reply:
Fixed